src/Stryker.Core/Stryker.Core/Stryker.Core.csproj

```diff
@@ -32,6 +32,8 @@
     <EmbeddedResource Include="Reporters\HtmlReporter\Files\mutation-report.html" />
 
     <EmbeddedResource Include="ToolHelpers\.vstest\Microsoft.TestPlatform.Portable\*\*.nupkg" />
+
+    <EmbeddedResource Include="microsoft.testplatform.portable.16.8.3.nupkg" />
```

Comment:
We already embed this just above this line. Why do you embed it again for a specific version number?